neo/rawio/intanrawio.py

```diff
@@ -530,6 +530,26 @@ def _demultiplex_digital_data(self, raw_digital_data, channel_ids, i_start, i_st
             output[:, channel_index] = demultiplex_data[i_start:i_stop].flatten()
 
         return output
+    
+    def get_intan_timestamps(self):
+        """
+        
+        Retrieves the timestamps from the Intan raw data based on the file format.
```

Comment:
```suggestion
        Retrieves the timestamps (which are in sample indices) from the Intan raw data.
        
```
I would say we basically add the note from below in a Notes section, but as a user of the function I shouldn't care in the simple docstring about file format I should just care about the timestamps no? 

For me I want to make it clear to the user that these are in samples and not seconds. So I want to highlight specifically that it is sample time in the main one-liner.